lib/modules/manager/helmsman/extract.ts

```diff
@@ -31,6 +32,14 @@ function createDep(
   }
   dep.currentValue = anApp.version;
 
+  // in case of OCI repository, we need a PackageDependency with a DockerDatasource and a packageName
+  const isOci = anApp?.chart?.startsWith('oci://');
+  if (isOci) {
+    dep.datasource = DockerDatasource.id;
+    dep.packageName = anApp.chart?.replace('oci://', '');
```

Comment:
```suggestion
    dep.packageName = anApp.chart!.replace('oci://', '');
```
we know it's there here